### DISCUSSION TITLE
Godot 4 needs a new Logo, you are confusing us!

### DISCUSSION THREAD

**Discussion by WhalesState** [godotengine/godot-proposals#7518] [General Discussions]
Can you please tell which one is godot 3 and which one is godot 4 ?

I think it's clear now. Thank you!

![image](https://github.com/godotengine/godot-proposals/assets/53877170/3a07372e-8ee7-4ed4-940e-1235adbcb7c7)

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/541.

I don't think we should change the logo just to distinguish between major versions. Instead, I suggest pinning only the version you use the most to your task bar (or not pinning any of them like I do). Add both Godot versions to your Start Menu with different names, and launch them using the Start Menu.

Windows also allows you to override an executable icon if I'm not mistaken, so you could use that as well.

**Comment by WhalesState** [comment]
ok just wanted to make sure that no one will even notice that both are godot 4, and yes a small change for icon is welcomed, like a letter c or # somewhere on the logo for mono version or the number 3 for the godot 3 version to keep the new godot 4 logo the same without confusion.

<del> @AThousandShips I don't know why the umm emoji for everything I post and now a thump down, even you replied in a bad way many times and i reply good all the time, stop doing that to not give a bad background about the Godot devs ;). </del>

**Reply by AThousandShips** [reply]
Thumbs down is a downvote, am I not allowed to dislike something?

You need to work on your ability to handle criticism, you read malice and bad behaviour into things where none exist, please act professionally, and don't exaggerate, you are behaving as if I am personally attacking you, and constantly reading things into what I say that I have not said or implied, this is not okay

But maybe I should have been more attentive to your struggles with this and avoided expressing my opinion as you are unable to handle that

**Reply by Zireael07** [reply]
@WhalesState AThousandShips is allowed to have her own opinion and her opinion is NOT the opinion of all the Godot devs.

Also you're reading too much into a thumbs down or an emoji imho.

**Reply by WhalesState** [reply]
I was talking about her bad replies to me two times not the emoji, i can take screenshots and show u what i mean but i'm not a fan of wasting time.

**Reply by AThousandShips** [reply]
Please, we already discussed our interactions previously and I asked you to express your concerns and issues with me and you did not respond and expressed that there were no issues

If you have issues with me take them with me directly, but please follow the [code of conduct](https://godotengine.org/code-of-conduct/) and act professionally, if you are not interested in responding when I ask you about what you would like me to change in my way of talking to you, as I asked already previously, then you need to stop accusing me or talking about me in this way, it is unprofessional

**Reply by WhalesState** [reply]
I don't have time now to read and reply more because I'm not home and am using phone, but to explain what i mean, this is when i was asking for a help fixing an issue, then she replied with this! I don't take anything personally and I'm here to help and learn only. Thanks to Calinou on his informative reply btw and to correct the mistake in her reply, but she could have replied in a better way, i can give more examples to her replies but as i said i don't like wasting time, just reply in a good way to the new devs and correct their mistakes nicely same as how Calinou did to correct your mistake, thank you.

![Screenshot_20230817-154656_GitHub.jpg](https://github.com/godotengine/godot-proposals/assets/53877170/62e19dab-b9c3-48c3-813b-503b7bcab1dd)

**Reply by Calinou** [reply]
@WhalesState Please don't derail this discussion with off-topic comments, or this discussion will have to be locked.

**Reply by AThousandShips** [reply]
You keep reading things into my comments that aren't there, please stop assuming intent where none exists, it's inappropriate and unprofessional, see the code of conduct, me being wrong isn't a "bad" comment, and me saying in a nice tone that I didn't have the time to dig through the code for you isn't either, I have a lot to do here

Have a nice day

Edit, adding as response:
I am sorry that it has felt that way, but you actively rejected the option to express this and work it out with me, which is on you, and you have a lot to work with in regards to taking criticism and handling how you react to things.

You will meet plenty of people in life that don't communicate in ways that you expect, and if you continue to respond to them based on what *you* assume, instead of what they intend, you are going to continue to experience issues, despite no one else being to blame but yourself.

**Reply by WhalesState** [reply]
Have a nice day too, it's ok i have seen worst already, just need to tell that it wasn't easy for me to be here and to contribute to the engine, I'm having back and neck pain everyday for 3 years while I'm still 27 so be easy on the new contributers because you don't know how hard they try to help in the growth of godot and also i was about to stop contributing to godot because of the way you reply. Mark this as the last off topic message. THANK YOU 😊
